pkg/cmd/cli/install/install.go

```diff
@@ -95,13 +109,19 @@ func (o *InstallOptions) AsVeleroOptions() (*install.VeleroOptions, error) {
 	if err != nil {
 		return nil, err
 	}
+	veleroPodResources, err := parseResourceRequests(o.VeleroPodCPURequest, o.VeleroPodMemRequest, o.VeleroPodCPULimit, o.VeleroPodMemLimit)
```

Comment:
parseResourceRequests takes parameters instead of simply getting them from the struct because it will need to be used for parsing restic pod resources too